Sources/Sentry/SentrySDK.m

```diff
@@ -698,6 +699,15 @@ + (void)stopProfiler
 }
 #endif // SENTRY_TARGET_PROFILING_SUPPORTED
 
+#if SENTRY_HAS_UIKIT
+
+/** Only needed for testing */
```

Comment:
`m`: If only used for testing, can we use `#if SENTRY_TEST || SENTRY_TEST_CI`?

Reply:
Nope sadly not. I added a comment. Sorry, could have done that.